pennylane/tape/interfaces/functions.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```

Comment:
For now this file is just supposed to collect all wrapper functions that seem absolutely necessary.